Examples/Python/src/MLTrackFindingStub.cpp

```diff
@@ -0,0 +1,18 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2023 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include "Acts/Plugins/Python/Utilities.hpp"
+
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+
+namespace Acts::Python {
+void addMLTrackFinding(Context& /*unused*/) {
```

Comment:
Since this does nothing maybe it should return an error if called ?

Reply:
this will actually still be called but only if `ML` is not present. in this case nothing will be added to the python bindings

Reply:
That was what I understood. But if you are calling `addMLTrackFinding` without having onnx installed you are probably making a mistake and there should probably be a message warning you.

Reply:
currently it is actually intended that it is called and does nothing because here https://github.com/acts-project/acts/blob/main/Examples/Python/src/ModuleEntry.cpp#L270 we use that to our advantage. we call the add function and either the module is present and will go through the binding stuff or it is not present and will not do anything

Reply:
I didn't know that. Make sense !